curriculum/challenges/english/02-javascript-algorithms-and-data-structures/intermediate-algorithm-scripting/sum-all-primes.english.md

```diff
@@ -8,9 +8,14 @@ forumTopicId: 16085
 
 ## Description
 <section id='description'>
-Sum all the prime numbers up to and including the provided number.
-A prime number is defined as a number greater than one and having only two divisors, one and itself. For example, 2 is a prime number because it's only divisible by one and two.
-The provided number may not be a prime.
+
+A <dfn>prime number</dfn> is a whole number greater than 1 with exactly two divisors: 1 and
+itself. For example, 2 is a prime number because it is only divisible by 1 and 2. In
+contrast, 4 is not prime since it is divisible by 1, 2 and 4.
+
+Rewrite `sumPrimes` so it returns the sum of all prime numbers that are less than or
+equal to num.
+
 Remember to use <a href='http://forum.freecodecamp.org/t/how-to-get-help-when-you-are-stuck/19514' target='_blank'>Read-Search-Ask</a> if you get stuck. Try to pair program. Write your own code.
```

Comment:
I would move this to the Instructions section.  Also, I would remove the extra line before and after the description.

Reply:
I'm not sure what extra lines you are referring to.

I initially put these in the instructions, but noticed that all of the other algorithms had everything in the description area and it didn't look right to me. I suggested that we should make an issue to fix all the algorithms at once, instead of this be the only one split into description and instructions. Don't you think it would be better to keep them consistent with each other?

Reply:
I hadn't thought about keeping them consistent.  Yeah, I think it would be better to do them all at once.

Reply:
Also, the extra lines make it so this is treated as markdown.  Removing them would stop the backticks working and mess with the formatting.